config/initializers/rack_cors.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins 'https://solidemo-graphql-playground.herokuapp.com'
```

Comment:
Thoughts on tying this to an `ENV` variable in true heroku fashion? 
The default value can be the current one if needed or turned into `https://demo.solidus.io`.

Reply:
Makes total sense @elia. Changed!